### PR TITLE
feat(bpm manage): BPM流程管理开发

### PR DESCRIPTION
**Pull request by guo-zi-xin**
feat(bpm manage): BPM流程管理开发
2023-05-09 16:12

### CODE REVIEW

**Comment by openc3-bot-001** [comment]
test/err

build-c3-front:err
check-yml-file-format:ok
compiler-bl-proxy:ok
compiler-jumpserver:ok
compiler-perl:err
compiler-trouble-ticketing:ok

File: c3-front/src/app/pages/business/bpm/bpm.controller.js [hunk 1]
```diff
@@ -65,6 +65,34 @@
...
+            showCancelButton: true,
+            confirmButtonColor: "#DD6B55",
+            cancelButtonText: "取消",
+            confirmButtonText: "确定",
+            closeOnConfirm: true
+          }, function(){
+            $http.post(`/api/job/bpm/manage/show/${name}/${Number(!+show)}`).success(function(data){
+              if (data.stat){
+                vm.reload();
+                swal({ title: '删除成功', type: 'success' });
```
Comment (lijinfeng2011) [inline c3-front/src/app/pages/business/bpm/bpm.controller.js]: 这里应该改成“执行成功”

Comment (lijinfeng2011) [reply c3-front/src/app/pages/business/bpm/bpm.controller.js]: @guo-zi-xin

**Comment by lijinfeng2011** [comment]
代码构建失败：

events.js:182
      throw er; // Unhandled 'error' event
      ^

Error: StringMap expected string key
    at stringmap.set (/code/node_modules/stringmap/stringmap.js:99:19)
    at Scope.add (/code/node_modules/ng-annotate/build/es5/scope.js:102:17)
    at /code/node_modules/ng-annotate/build/es5/scopetools.js:38:25
    at Array.forEach (<anonymous>)
    at createScopes (/code/node_modules/ng-annotate/build/es5/scopetools.js:36:27)
    at visit (/code/node_modules/ordered-ast-traverse/ordered-ast-traverse.js:25:19)
    at visit (/code/node_modules/ordered-ast-traverse/ordered-ast-traverse.js:43:25)
    at visit (/code/node_modules/ordered-ast-traverse/ordered-ast-traverse.js:46:21)
    at visit (/code/node_modules/ordered-ast-traverse/ordered-ast-traverse.js:43:25)
    at visit (/code/node_modules/ordered-ast-traverse/ordered-ast-traverse.js:46:21)
    at visit (/code/node_modules/ordered-ast-traverse/ordered-ast-traverse.js:43:25)
    at visit (/code/node_modules/ordered-ast-traverse/ordered-ast-traverse.js:46:21)
    at visit (/code/node_modules/ordered-ast-traverse/ordered-ast-traverse.js:43:25)
    at visit (/code/node_modules/ordered-ast-traverse/ordered-ast-traverse.js:46:21)
    at visit (/code/node_modules/ordered-ast-traverse/ordered-ast-traverse.js:46:21)
    at visit (/code/node_modules/ordered-ast-traverse/ordered-ast-traverse.js:46:21)

File: c3-front/src/app/pages/business/bpm/bpm.controller.js (hunk 1 above)
Comment (guo-zi-xin) [reply c3-front/src/app/pages/business/bpm/bpm.controller.js]: 已经修改

**Comment by guo-zi-xin** [comment]
打包失败原因是由于用了Es6的解构语法 目前项目不支持 已经修改Es5写法

**Pull request merged**